### DISCUSSION TITLE
Trigger ClickEvent when annotation is not visible

### DISCUSSION THREAD

**Discussion by UteSpiske** [annotorious/annotorious#351] [Q&A] [answered]
Hei, 

I want to see if a user clicks inside an annotation. This works perfectly when the annotation is visible with 
 ```
anno.on('clickAnnotation', function (){
        console.log('Annotation was clicked !');
    });
```
Is there a way to do the same when the visibility is set to false ?

**Comment by rsimon** [comment, marked as answer]
Hi,

definitely not in Annotorious 2.x I'm afraid. (If I remember correctly, hiding annotations is done by removing the whole annotation layer, including all event listeners.) It could be a feature to put on the list for the upcoming v3, though.
